src/js/page.js

```diff
@@ -145,9 +147,9 @@
       let nodeSelector = "#newtab-grid, #newtab-search-container";
       for (let node of document.querySelectorAll(nodeSelector)) {
         if (aValue) {
```

Comment:
I wonder if we should rename "aValue", just to give more context of what it is in these kinds of situations.